src/remote/remote.go

```diff
@@ -708,7 +708,8 @@ func (c *Client) updateProgress(tid int, target *core.BuildTarget, metadata *pb.
 		case pb.ExecutionStage_EXECUTING:
 			c.state.LogBuildResult(tid, target.Label, core.TargetBuilding, "Building...")
 		case pb.ExecutionStage_COMPLETED:
-			c.state.LogBuildResult(tid, target.Label, core.TargetBuilt, "Built")
+			// The build result will be logged in build/build_step.go
```

Comment:
Might still want to log this as `TargetBuilding` but with a slightly different message?

Reply:
Oh maybe, I figured it was moments away from returning back to build_step.go so it didn't really matter. Though ti would be a bit weird to log "Built" with the status `TargetBuilding` no?

Reply:
Yeah Complete sounds good (I was thinking "Execution complete" but shorter is better)